packages/tailwind-constructor/tailwind.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = withTV({
       transparent: 'transparent',
       ...filterTokensByType('color', tokens),
     },
+    extend: {
+      colors: {
+        'sky-blue': '#7dd3fc',
+      }},
```

Comment:
@tiwarineelu59 why are you adding this new color? this needs to be approved by design team.

@Nouri-Anouar please check if this should be added or not

Reply:
@Enjoy2Live This color was used in the user profile design layout for one of the texts but was not able to apply so I had to add this in tailwind.confi.js

Reply:
Hello @tiwarineelu59 
Could you please provide me the Figma link where this color is being used?

Reply:
@tiwarineelu59 just making sure you are receiving these notifications, if you do sorry for the additional ping!

Reply:
@Nouri-Anouar https://www.figma.com/design/D31z19f4we75VkmEGzdyUn/User-Profile-%2F-Onboarding-Main-File-(Copy)?node-id=2239-18693&m=dev